doc/api/os.md

```diff
@@ -1160,6 +1160,52 @@ The following error codes are specific to the Windows operating system:
   </tr>
 </table>
 
+### dlopen Constants
+
+If available on the operating system, the following signal constants
+are exported by `os.constants.dlopen`. See [man 3 dlopen][] for detailed
```

Comment:
Also, the way man page links are handled is to use the format like 'dlopen(3)' and it will get automatically linked when the markdown is processed.